client/src/app/project-registration/project-registration-form/project-registration-form.component.ts

```diff
@@ -226,4 +234,25 @@ export class ProjectRegistrationFormComponent implements OnInit {
         concatMap(createdProject => this.ingestService.partiallyPatchProject(createdProject, this.patch))
       );
   }
+
+  onFormValueChange(formData: Observable<object>) {
+    formData.pipe(
+      delay(10000), // 10 seconds
```

Comment:
any comments / feedback on what the delay should be or if 10 seconds is okay?

This can also be made different for the different environments, if needed

Reply:
better to have this in environment config